ci-bench-runner/Cargo.toml

```diff
@@ -6,17 +6,28 @@ edition = "2021"
 [dependencies]
 anyhow = "1.0.75"
 axum = "0.6.20"
+bencher_client = { git = "https://github.com/bencherdev/bencher", rev = "d0532976" }
```

Comment:
Are there any plans for a crates.io release? Depending on a GitHub repository is less friendly to reproducible builds

Reply:
Definitely! That has been on my todo (https://github.com/bencherdev/bencher/issues/66). If it is a blocker here or even just a nice to have, I can prioritize getting it done.

Pinning things to a `rev` should keep them reproducible. Though, I definitely understand the preference for a crates.io release.

Reply:
I agree that a `rev` helps, and I guess the repo won't disappear anytime soon, but it's good to know a crates.io release is coming at some point